docs/integrations/monitor-non-service-metrics-prometheus.rst

```diff
@@ -0,0 +1,101 @@
+
+
+CPU usage 
+----------
+CPU usage metrics are helpful in determining if the CPU is constantly being maxed out.
+For a high-level view of CPU usage for a single CPU service, you can use
+
+.. code:: 
+  
+    100 - cpu_usage_idle{cpu="cpu-total"}
+
+Bear in mind that it may not be a good idea to use ``cpu_usage_system{cpu="cpu-total"}`` + ``cpu_usage_user{cpu="cpu-total"}`` to calculate total CPU usage, because they do not include ``cpu_usage_iowait``. Furthermore, a process with a ``nice`` value larger than 0 will be categorized as ``cpu_usage_nice``, which is not included in ``cpu_usage_user``.
+
+It is also worth monitoring ``cpu_usage_iowait{cpu="cpu-total"}``. If this number is high, it indicates the service node is working on something I/O intensive. For example, 40 means 40% of CPU time is waiting for disk or network I/O.
+
+The following table lists some important CPU-related metrics you can collect and monitor:
+
+
+.. list-table::
+  :header-rows: 1
+  :align: left
+
+  * - Metics
+    - Description
+  * - ``cpu_usage_idle``
+    - The percentage of time the CPU is ideal. 
+  * - ``cpu_usage_system``
+    - The percentage of time the Kernel code is consuming the CPU. 
+  * - ``cpu_usage_user``
+    - The percentage of time the CPU is in user-space program with nice value <= 0.
+  * - ``cpu_usage_nice``
+    - The percentage of time the CPU is in user-space program with nice value > 0.
+  * - ``cpu_usage_iowait``
+    - The percentage of time waiting for I/O operations. 
+  * - ``cpu_usage_steal``
+    - The percentage of time waiting for the hypervisor to give CPU cycles to the VM. 
+  * - ``cpu_usage_irq``
+    - The percentage of time the system is handling interrupts. 
+  * - ``cpu_usage_softirq``
+    - The percentage of time the system is handling software interrupts. 
+  * - ``cpu_usage_guest``
+    - The percentage of time the CPU is running for a guest OS.
+  * - ``cpu_usage_guest_nice``
+    - The percentage of time the CPU is running for a guest OS, with low priority.
+
+These metrics are generated from the `telegraf plugin <https://github.com/influxdata/telegraf/tree/master/plugins/inputs/cpu>`_.
+
+Disk usage
+----------
+
+Monitoring disk usage ensures that applications or processes will not fail due to insufficient disk storage. Consider monitoring ``disk_used_percent`` and ``disk_free``.
+
+The following table lists some important disk usage metrics you can collect and monitor:
+
+.. list-table::
+  :header-rows: 1
+  :align: left
+
+  * - Metics
+    - Description
+  * - ``disk_free``
+    - Free space on service disk. 
+  * - ``disk_used``
+    - Used space on disk. For example, 8.0e+9 means 8,000,000,000 bytes.   
+  * - ``disk_total``
+    - The total space on disk (free and used space). 
+  * - ``disk_used_percent``
+    - The percentage of the disk space used. The is equal to ``disk_used ``/ ``disk_total`` * 100. For example, 80 means 80% service disk usage. 
```

Comment:
This has ended up weirdly rendered. I think we could make the whole calculation literal: `disk_used / disk_total * 100`

Reply:
Fixed